src/components/flow/ui/TypedInput.tsx

```
@@ -33,33 +41,59 @@ interface TypedInputProps {
+		case TypedInputEnum.Parens:
+			return {
+				type: TypedInputEnum.Parens,
+				inner: { type: TypedInputEnum.Nil }
+			}; // not supported in 'simple' UI outside ComplexSubflow
+		case TypedInputEnum.LogicExpr:
+			return {
+				type: TypedInputEnum.LogicExpr,
+				condition: TypedInputLogicType.And,
+				operands: [{ type: TypedInputEnum.Nil }]
+			}; // not supported in 'simple' UI outside ComplexSubflow
+		case TypedInputEnum.Vector:
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)